### ISSUE TITLE
Video loads, but annotations disappear on develop

### ISSUE THREAD

**Issue by absempere** [AVAnnotate/project-client#82]
I’m on the develop branch and annotations aren’t staying on this page (manifest we’ve used before, created a new project on Friday on staging/develop): https://avannotate.github.io/dev-demo/events/readings-and-discussion-featuring-ualberta-wr/

Annotations load in the presentation view and then disappear as though suppressed.

**Comment by camdendotlol** [comment]
@absempere Would you be able to record your screen when this happens? I still can't seem to replicate this on Firefox or Chrome.

**Comment by camdendotlol** [comment]
@absempere Could you open your browser console (fn + F12, or Cmd + Shift + C) and then take another recording? I was able to replicate the bug once yesterday after dozens of reloads but I can't seem to get it again today.

**Comment by camdendotlol** [comment]
I see this error occasionally but I think it's unrelated and not critical. The annotations don't disappear when just this error happens.

<img width="317" alt="Screenshot 2024-10-29 at 9 44 48 AM" src="https://github.com/user-attachments/assets/531378a3-9164-4422-bacc-9a7ee7a34a1d">

The one time I replicated the above issue, I saw something about `.length` and `useMemo`, in addition to the error in my screenshot.
